src/test/java/subway/StationAcceptanceTest.java

```diff
@@ -55,13 +59,81 @@ void createStation() {
      * When 지하철역 목록을 조회하면
      * Then 2개의 지하철역을 응답 받는다
      */
-    // TODO: 지하철역 목록 조회 인수 테스트 메서드 생성
+    @DisplayName("지하철역을 조회한다.")
+    @Test
+    void getStationsSuccess() {
+        // given
+        final var createdStations = List.of(
+            this.createStation("선릉역"),
+            this.createStation("청계산입구역")
+        );
+
+        // when
+        final var response = RestAssured
+            .given().log().all()
+            .contentType(MediaType.APPLICATION_JSON_VALUE)
+            .when().get("/stations")
+            .then().log().all()
+            .extract();
+
+        // then
+        assertThat(response.statusCode()).isEqualTo(HttpStatus.OK.value());
+
+        // then
+        assertThat(response.jsonPath().getList("id", Long.class))
+            .containsAll(
+                createdStations.stream()
+                    .map(StationResponse::getId)
+                    .collect(Collectors.toList())
+            );
+    }
 
     /**
      * Given 지하철역을 생성하고
      * When 그 지하철역을 삭제하면
      * Then 그 지하철역 목록 조회 시 생성한 역을 찾을 수 없다
      */
-    // TODO: 지하철역 제거 인수 테스트 메서드 생성
+    @DisplayName("지하철역을 삭제한다.")
+    @Test
+    void deleteStationSuccess() {
+        // given
+        final var deletedStation = this.createStation("삼성역");
+
+        // when
+        RestAssured
+            .given().log().all()
+            .contentType(MediaType.APPLICATION_JSON_VALUE)
+            .when().delete("/stations/{id}", deletedStation.getId())
+            .then().log().all()
+            .extract();
+
+        final var responseStations = getStations();
+
+        // then
+        assertThat(responseStations.stream()
+            .filter(responseStation -> deletedStation.getId().equals(responseStation.getId()))
+            .findAny()
+            .isEmpty()
+        ).isTrue();
```

Comment:
```suggestion
                assertThat(deletedStation.getId()).isNotIn(responseStations);
```
요런 방식으로 좀 더 간결히 표현해볼 수 있습니다! assertThat뒤에 오는 많은 assertion함수들이 있는데, 이번 과정 진행해보시면서 여러가지 사용해보시는 것도 추천드려요!

Reply:
오 좋네요! 감사합니다